app/controllers/events_controller.rb

```diff
@@ -11,8 +11,20 @@ def index
     end
     @all_events = Event.all.order(start_time: :asc)
     @events = Event.where("end_time > ?", Time.now).order(start_time: :asc).paginate(page: params[:page], per_page: 10)
+    
+    respond_to do |format|
+      format.html
+      format.csv { 
+        if current_user.is_admin?  
```

Comment:
can you standardize on `current_user.is_admin? or current_user.is_organizer?` like you did below